generator/discovery/openstack/vm_images_discovery.py

```diff
@@ -46,7 +39,7 @@
     "openrisc": "RISC-V",
     "parisc": "RISC-V",
     "parisc64": "RISC-V",
-    "unicore32": "RISC-V",
+    "unicore32": "RISC-V"
```

Comment:
```suggestion
    "unicore32": "RISC-V",
```

Why was the trailing comma removed?

Reply:
Because it's the last element of dict and comma is not required.

Reply:
https://peps.python.org/pep-0008/#when-to-use-trailing-commas

> When trailing commas are redundant, they are often helpful when a version control system is used, when a list of values, arguments or imported items is expected to be extended over time. The pattern is to put each value (etc.) on a line by itself, always adding a trailing comma, and add the close parenthesis/bracket/brace on the next line.

Reply:
How is this resolved?

Reply:
Done.